api/cluster-resources/setup_notebook_cluster.sh

```diff
@@ -37,13 +37,14 @@ echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 # base Leo image or a custom AoU docker image.
 apt-get update
 apt-get -t stretch-cran35 install -y --no-install-recommends libmagick++-dev gfortran-6
+apt-get install -y --no-install-recommends jq
 
-for v in "2.7" "3"; do
+for v in "3"; do
```

Comment:
should we just drop the for loop since there's only 1 element now?

Reply:
I thought about that, sure. Sounds reasonable.